airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2110,6 +2110,16 @@ components:
           format: uri
         icon:
           type: string
+        releaseStage:
+          type: string
+          enum:
+            - alpha
+            - beta
+            - generally_available
+            - custom
+        releaseDate:
+          description: The date when this connector was first released, in yyyy-mm-dd format.
```

Comment:
Should we use the `format: "date"` option for this (see https://swagger.io/docs/specification/data-models/data-types/)?